alb.tf

```diff
@@ -38,6 +38,18 @@ resource "aws_alb_target_group" "main" {
     }
   }
 
+  dynamic "stickiness" {
+    for_each = length(keys(lookup(var.target_groups[count.index], "stickiness", {}))) == 0 ? [] : [
+    lookup(var.target_groups[count.index], "stickiness", {})]
+
+    content {
+      cookie_duration = lookup(stickiness.value, "cookie_duration", null)
+      cookie_name     = lookup(stickiness.value, "cookie_name", null)
+      enabled         = lookup(stickiness.value, "enabled", null)
+      type            = lookup(stickiness.value, "type", null)
```

Comment:
guess we don't know the LB type here to prefil for NLBs like in https://github.com/terraform-aws-modules/terraform-aws-alb/blob/7ed6f047d37510f60c866efc984757fc6b660007/main.tf#L479, do we?

Reply:
I am not totally sure on this one.